### DISCUSSION TITLE
How to reenable move to next file difference

### DISCUSSION THREAD

**Discussion by commanderkoo** [WinMerge/winmerge#517] [Q&A] [answered]
Hello, when you start Winmerge for the first time, and you go past the last diff of a file, there's a dialog that asks if you want to move to the first difference of the next file.  

Is there any way to reenable that dialogue after specifying "don't show this dialogue again"

Thanks!

**Comment by sdottaka** [comment, marked as answer]
To make the message box reappear, press the Reset button on the General page of the Options dialog.

![image](https://user-images.githubusercontent.com/98126/102829722-11550880-442b-11eb-8a66-77450b6db45d.png)

**Comment by sdottaka** [comment]
In WinMerge 2.16.14, the reset button has been moved to the Message Boxes category. 

![image](https://user-images.githubusercontent.com/98126/133929080-89f5d165-a6f6-492a-8262-ec6e37d64ac8.png)
